dbt/models/model/schema.yml

```diff
@@ -4,61 +4,192 @@ sources:
     tables:
       - name: assessment_card
         description: '{{ doc("table_assessment_card") }}'
+        data_tests:
+          - unique_combination_of_columns:
+              name: model_assessment_card_unique_by_pin_card_and_year
+              combination_of_columns:
+                - meta_pin
+                - meta_card_num
+                - meta_year
+                - run_id
+              config:
+                  error_if: ">5748"
+              meta:
+                description: assessment card should be unique by pin, card, year, and run_id
```

Comment:
**suggestion:** I don't think we actually need a description for these tests, since that's mainly used as an input for constructing Excel QC workbooks. Since these tests are exclusive to GitHub Actions (i.e. are only data tests), the description isn't needed.